orchagent/aclorch.cpp

```diff
@@ -184,7 +184,23 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
                     SWSS_LOG_ERROR("Failed to locate port %s", alias.c_str());
                     return false;
                 }
-                m_inPorts.push_back(port.m_port_id);
+                if (port.m_type == Port::PHY)
+                {
+                    m_inPorts.push_back(port.m_port_id);
+                }
+                else if (port.m_type == Port::LAG)
```

Comment:
this does not match the SAI spec. only type PORT is support for SAI_ACL_ENTRY_ATTR_FIELD_IN_PORTS

https://github.com/opencomputeproject/SAI/blob/master/inc/saiacl.h#L1247

Reply:
Thinks, we will delete LAG port.